lib/node_modules/@stdlib/blas/ext/base/sdssum/manifest.json

```diff
@@ -1,42 +1,180 @@
 {
-	"options": {},
-	"fields": [
-		{
-			"field": "src",
-			"resolve": true,
-			"relative": true
-		},
-		{
-			"field": "include",
-			"resolve": true,
-			"relative": true
-		},
-		{
-			"field": "libraries",
-			"resolve": false,
-			"relative": false
-		},
-		{
-			"field": "libpath",
-			"resolve": true,
-			"relative": false
-		}
-	],
-	"confs": [
-		{
-			"src": [
-				"./src/sdssum.c"
-			],
-			"include": [
-				"./include"
-			],
-			"libraries": [
-				"-lm"
-			],
-			"libpath": [],
-			"dependencies": [
-				"@stdlib/blas/ext/base/sdssumpw"
-			]
-		}
-	]
-}
+  "options": {
+    "os": "linux",
+    "blas": "",
+    "wasm": false
+  },
+  "fields": [
+    {
+      "field": "src",
+      "resolve": true,
+      "relative": true
+    },
+    {
+      "field": "include",
+      "resolve": true,
+      "relative": true
+    },
+    {
+      "field": "libraries",
+      "resolve": false,
+      "relative": false
+    },
+    {
+      "field": "libpath",
+      "resolve": true,
+      "relative": false
+    }
+  ],
+  "confs": [
+    {
+      "os": "linux",
+      "blas": "",
+      "wasm": false,
+      "src": [
+        "./src/sdssum.f",
+        "./src/sdssumsub.f",
+        "./src/sdssum_f.c"
+      ],
+      "include": [
+        "./include"
+      ],
+      "libraries": [],
+      "libpath": [],
+      "dependencies": [
+        "@stdlib/napi/export",
+        "@stdlib/napi/argv",
+        "@stdlib/napi/argv-int32",
+        "@stdlib/napi/argv-strided-float32array"
+      ]
+    },
+    {
+      "os": "linux",
+      "blas": "openblas",
+      "wasm": false,
+      "src": [
+        "./src/sdssum_cblas.c"
+      ],
+      "include": [
+        "./include"
+      ],
+      "libraries": [
+        "-lopenblas",
+        "-lpthread"
+      ],
+      "libpath": [],
+      "dependencies": [
+        "@stdlib/napi/export",
+        "@stdlib/napi/argv",
+        "@stdlib/napi/argv-int32",
+        "@stdlib/napi/argv-strided-float32array"
+      ]
+    },
+    {
+      "os": "mac",
+      "blas": "",
+      "wasm": false,
+      "src": [
+        "./src/sdssum.f",
+        "./src/sdssumsub.f",
+        "./src/sdssum_f.c"
+      ],
+      "include": [
+        "./include"
+      ],
+      "libraries": [],
+      "libpath": [],
+      "dependencies": [
+        "@stdlib/napi/export",
+        "@stdlib/napi/argv",
+        "@stdlib/napi/argv-int32",
+        "@stdlib/napi/argv-strided-float32array"
+      ]
+    },
+    {
+      "os": "mac",
+      "blas": "apple_accelerate_framework",
+      "wasm": false,
+      "src": [
+        "./src/sdssum_cblas.c"
+      ],
+      "include": [
+        "./include"
+      ],
+      "libraries": [
+        "-lblas"
+      ],
+      "libpath": [],
+      "dependencies": [
+        "@stdlib/napi/export",
+        "@stdlib/napi/argv",
+        "@stdlib/napi/argv-int32",
+        "@stdlib/napi/argv-strided-float32array"
+      ]
+    },
+    {
+      "os": "mac",
+      "blas": "openblas",
```

Comment:
There are no OpenBLAS bindings. This `manifest.json` is entirely incorrect. Use the former `manifest.json` file and just change the indentation.